app/services/content_metadata_generator.rb

```diff
@@ -77,6 +78,17 @@ def preserve_attr(cocina_file)
     cocina_file.fetch('administrative').fetch('sdrPreserve') ? 'yes' : 'no'
   end
 
+  def ocr?(cocina_file)
+    if cocina_file.fetch('hasMimeType') == 'text/html'
+      # finds file in @file_names, parses HTML with Nokogiri for hOCR tags to return
+      # true if found
+      file_path = file_names.fetch(cocina_file.fetch('filename'))
+      doc = Nokogiri::HTML(File.open(file_path))
+      return true unless doc.css("meta[@name='ocr-capabilities']").empty?
```

Comment:
Based on http://kba.cloud/hocr-spec/1.2/#metadata, the `ocr-capabilities` metadata element must be present for hOCR. We could also use `ocr-system` metadata element as well.